packages/database/src/migrations/20210503020225-LESMISReportsNewPermissionGroup-modifies-data.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const permissionGroupNameToId = async (db, name) => {
+  const record = await db.runSql(`SELECT id FROM permission_group WHERE name = '${name}'`);
```

Comment:
```suggestion
  const result = await db.runSql(`SELECT id FROM permission_group WHERE name = '${name}'`);
```
It's not really a record at this point